### DISCUSSION TITLE
new Release to fix problem in .Net 8

### DISCUSSION THREAD

**Discussion by DaveLaa** [apache/logging-log4net#114] [General]
Since I updated my project to .Net 8 my logging is not working properly. It is not showing the thread number in my massive multithreading service and this way it is almost impossible to find, which log threads belong together.

The fix of Eric Mavrinac is already checked in in git in early january and I wait for this fix to be played out in a new release.

Is there a date, when a new release version of Log4Net is planned? The last one was 1,5 years ago and atm Log4Net is not really usable in the new .Net 8.

**Comment by FreeAndNil** [comment]
Hi @DaveLaa we've released https://www.nuget.org/packages/log4net/2.0.16
